data/partials/requests.yaml

```diff
@@ -1,49 +1,54 @@
     link: "/infrastructure/process"
     image: "int_processes.png"
-  - name: "Files / Directories"
-    description: "Measuring the age of files, the number of files in a directory, or the size of a directory can be accomplished with the Directory check."
+  - name: "Files and Directories"
+    description: The Directory check measures the age of files, the number of files in a directory, or the size of a directory."
     link: "/integrations/directory"
     image: "int_files.png"
+  - name: "Endpoint"
+    description: "Use the Agent-based [HTTP check](https://docs.datadoghq.com/integrations/http_check/), or configure [Synthetic Monitoring](https://docs.datadoghq.com/synthetics/) from the Datadog application to validate if an endpoint or URL is running and accessible. Use either option to test both public and private endpoints. Combine with [Service Level Objectives, or SLOs](https://docs.datadoghq.com/monitors/service_level_objectives/) to define clear targets for performance."
+    link: "/synthetics"
+    image: "int_endpoint.png"
+  - name: "SNMP and Network Traffic"
+    description: "[Network Device Monitoring](https://docs.datadoghq.com/network_monitoring/devices/) enables you to collect [SNMP](https://docs.datadoghq.com/integrations/snmp/) (Simple Network Management Protocol) metrics emitted from network devices, such as routers, switches, and printers. [Network Performance Monitoring](https://docs.datadoghq.com/network_monitoring/performance/) tracks all network traffic in and out of a host, providing visibility into your network traffic between services, containers, availability zones, and any other tag in Datadog. Connection data at the IP, port, and PID levels is aggregated into application-layer dependencies between meaningful `source` and `destination` endpoints."
+    link: "/network_monitoring"
+    image: "int_network.png"
   - name: "Cloud Providers"
-    description: "All the major Cloud providers (AWS, GCP, Azure, Alibaba) emit metrics through APIs. Use the Datadog integration tiles in your account to configure these integrations which use Datadog servers to crawl and store your metrics."
+    description: "
+All the major Cloud providers ([AWS](https://docs.datadoghq.com/integrations/amazon_web_services/), [Azure](https://docs.datadoghq.com/integrations/azure/), [GCP](https://docs.datadoghq.com/integrations/google_cloud_platform/), [Alibaba](https://docs.datadoghq.com/integrations/alibaba_cloud/)) emit metrics through APIs. Use the [Datadog integration tiles](https://app.datadoghq.com/integrations) in your account to configure these integrations, which use Datadog servers to crawl for metrics."
     link: "/integrations/#cat-cloud"
     image: "int_cloud.png"
+  - name: "Windows Performance Counters"
+    description: "Windows performance counters can be collected with the PDH integration."
```

Comment:
Can we direct users to the Windows Performance Counters check. We dissuade PDH usage as seen here https://github.com/DataDog/integrations-core/tree/master/pdh_check